lib/node_modules/@stdlib/array/complex64/README.md

```diff
@@ -856,6 +856,82 @@ var count = context.count;
 // returns 3
 ```
 
+<a name="method-filter"></a>
+
+#### Complex64Array.prototype.filter( predicate\[, thisArg] )
+
+Returns a new array containing the elements of an array which pass a test implemented by a predicate function.
+
+```javascript
+var realf = require( '@stdlib/complex/realf' );
+var imagf = require( '@stdlib/complex/imagf' );
+var Complex64 = require( '@stdlib/complex/float32' );
```

Comment:
@Jaysukh-409 This doesn't appear to be used, correct?

Reply:
Yes, Sorry for that. We don't need the Complex64 here.

Reply:
@Jaysukh-409 No worries. If you could remove here and in other examples, that would be great!